### DISCUSSION TITLE
Backup audit data

### DISCUSSION THREAD

**Discussion by pgrange** [ACINQ/eclair#2841] [Q&A] [answered]
The [Backup section](https://github.com/ACINQ/eclair/blob/master/README.md#backup) of the README explains how to backup the channels states with eclair.sqlite.bak.

When not using PostgreSQL, what do you recommend to backup also _accounting_ data such as the ones returned with the [audit](https://acinq.github.io/eclair/#audit) API call? I guess this information is stored in _audit_ sqlite files but I've not found any recommandation comparable to the _.bak_ file and was wondering how to ensure data consistency with these files.

**Comment by t-bast** [comment, marked as answer]
There is no support for doing this with sqlite. The information in that DB isn't critical for node operation, and can thus be backed up "outside" of eclair (with no consistency guarantees). If you want strong consistency guarantees for that DB, you should probably be using postgres.

**Reply by pgrange** [reply]
Thank you for your answer @t-bast, it makes sense.

[litestream](https://litestream.io) seems to do the job to backup the audit database from outside eclair.

Just in case somebody else is interested, I've published a basic [ansible role](https://gitlab.com/21research/ansible/litestream) to install litestream on Debian for this kind of usage.
